report/mapping_functions.go

```diff
@@ -48,6 +48,37 @@ func ProcessPID(_ string, m NodeMetadata, grouped bool) (MappedNode, bool) {
 	}, show
 }
 
+// ProcessContainer maps Process topology nodes to the containers they run in.
+// We consider container and image IDs and names to be globally-scoped, and so
+// don't scope them further by e.g. host. If no container metadata is found,
+// nodes are grouped into the Uncontained node. If grouped is true, nodes are
+// grouped by container image.
+func ProcessContainer(_ string, m NodeMetadata, grouped bool) (MappedNode, bool) {
+	var (
+		containerID   = m["docker_id"]
+		containerName = m["docker_name"]
+		imageID       = m["docker_image_id"]
+		imageName     = m["docker_image_name"]
+		domain        = m["domain"]
+	)
+
+	var id, major, minor, rank string
+	if containerID == "" {
+		id, major, minor, rank = "uncontained", "Uncontained", "", "uncontained"
```

Comment:
So you went with uncontained....  

I think this will potentially end up a lot like the pseudo nodes if we're not careful, so it might be best to just drop them at this stage?